core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.html

```diff
@@ -2,8 +2,11 @@
 <ng-template #enabledComponent>
     <dotcms-template-builder
         [templateLayout]="pageState.layout"
+        (layoutChange)="onLayoutChange($event)"
```

Comment:
```suggestion
        (change)="onLayoutChange($event)"
```

Reply:
We can not use the `change` or `onChange` name. It throws an ESLint error: `Output bindings, including aliases, should not be named as standard DOM events`.

However, I can disable the rule for that `@Output`, what do you think, @fmontes?